internal/compiler/langtype.rs

```diff
@@ -64,6 +64,8 @@ pub enum Type {
 
     /// This is a `SharedArray<f32>`
     LayoutCache,
+    /// This is used by GridLayoutOrganizedData
+    ArrayOfU16,
```

Comment:
Not consistent that the `LayoutCache` name is describing the semantics of it with a comment saying how it is implemented, while `ArrayOfU16` is referring to the implementation with a comment explaining how it is used.

Reply:
I'm a bit torn.

The thing is that this `langtype::Type` enum was initially meant for the types that could be used in the language itself (property type, return value or arguments of callbacks).
But now it grew to also include all sort of "internal" types.

I don't have a strong opinion on whether the name should be semantic rather than how it is represented in the generated code. Just a bit annoying it is not consistent. Anyway, we can live with that so the patch is merged now.